cmake/multi_image.cmake

```
@@ -158,19 +158,24 @@ function(add_child_image_from_source)
   # | - prj.conf
   # | - prj_<desc>.conf
   # | - child_image DIRECTORY
-  #     |-- <ACI_NAME>.conf (I)             Fragment, used together with (A) and (C)
-  #     |-- <ACI_NAME>_<desc>.conf (J)      Fragment, used together with (B) and (D)
+  #     |-- <ACI_NAME>.conf (I)                 Fragment, used together with (A) and (C)
+  #     |-- <ACI_NAME>_<desc>.conf (J)          Fragment, used together with (B) and (D)
+  #     |-- <ACI_NAME>.overlay                  If present will be used to merge with BOARD.dts
   #     |-- <ACI_NAME> DIRECTORY
   #         |-- boards DIRECTORY
-  #         |   |-- <board>.conf (E)        If present, use instead of (C), requires (G).
-  #         |   |-- <board>_<desc>.conf (F) If present, use instead of (D), requires (H).
-  #         |-- prj.conf (G)                If present, use instead of (A)
-  #         |                               Note that (C) is ignored if this is present.
-  #         |                               Use (E) instead.
-  #         |-- prj_<desc>.conf (H)         If present, used instead of (B) when user
-  #                                         specify `-DCONF_FILE=prj_<desc>.conf for
-  #                                         parent image. Note that any (C) is ignored
-  #                                         if this is present. Use (F) instead.
+  #         |   |-- <board>.conf (E)            If present, use instead of (C), requires (G).
+  #         |   |-- <board>_<desc>.conf (F)     If present, use instead of (D), requires (H).
+  #         |   |-- <board>.overlay             If present will be used to merge with BOARD.dts
```

Consider 

```suggestion
  #         |   |-- <board>.overlay             If present, will be merged with BOARD.dts
```
here and for the other ones as well

---

Changed according to suggestion.